src/docfx/lib/markdown/MarkdownPipelineType.cs

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Collections.Generic;
-
 namespace Microsoft.Docs.Build
 {
-    internal class MarkupResult
+    public enum MarkdownPipelineType
     {
-        // TODO: remove this class
-        public List<Error> Errors = new List<Error>();
+        Markdown,
```

Comment:
here the markdown is conceptual markdown right? why rename it, we can have more markdown like overwrite markdown?